### ISSUE TITLE
build breaks when upgrading from v0.4.0 to v0.5.0

### ISSUE THREAD

**Issue by josharian** [GoogleCloudPlatform/berglas#99]
At v0.4.0, my project that uses berglas builds fine. At v0.5.0, I get:

```
2020/01/23 22:04:59 # github.com/GoogleCloudPlatform/berglas/pkg/berglas
../../../pkg/mod/github.com/!google!cloud!platform/berglas@v0.5.0/pkg/berglas/access.go:120:62: cannot use &secrets.AccessSecretVersionRequest literal (type *secrets.AccessSecretVersionRequest) as type *"google.golang.org/genproto/googleapis/cloud/secretmanager/v1beta1".AccessSecretVersionRequest in argument to c.secretManagerClient.AccessSecretVersion
../../../pkg/mod/github.com/!google!cloud!platform/berglas@v0.5.0/pkg/berglas/create.go:127:61: cannot use &secrets.CreateSecretRequest literal (type *secrets.CreateSecretRequest) as type *"google.golang.org/genproto/googleapis/cloud/secretmanager/v1beta1".CreateSecretRequest in argument to c.secretManagerClient.CreateSecret
../../../pkg/mod/github.com/!google!cloud!platform/berglas@v0.5.0/pkg/berglas/create.go:148:66: cannot use &secrets.AddSecretVersionRequest literal (type *secrets.AddSecretVersionRequest) as type *"google.golang.org/genproto/googleapis/cloud/secretmanager/v1beta1".AddSecretVersionRequest in argument to c.secretManagerClient.AddSecretVersion
../../../pkg/mod/github.com/!google!cloud!platform/berglas@v0.5.0/pkg/berglas/delete.go:110:52: cannot use &secrets.DeleteSecretRequest literal (type *secrets.DeleteSecretRequest) as type *"google.golang.org/genproto/googleapis/cloud/secretmanager/v1beta1".DeleteSecretRequest in argument to c.secretManagerClient.DeleteSecret
../../../pkg/mod/github.com/!google!cloud!platform/berglas@v0.5.0/pkg/berglas/list.go:146:47: cannot use &secrets.ListSecretsRequest literal (type *secrets.ListSecretsRequest) as type *"google.golang.org/genproto/googleapis/cloud/secretmanager/v1beta1".ListSecretsRequest in argument to c.secretManagerClient.ListSecrets
../../../pkg/mod/github.com/!google!cloud!platform/berglas@v0.5.0/pkg/berglas/list.go:184:55: cannot use &secrets.ListSecretVersionsRequest literal (type *secrets.ListSecretVersionsRequest) as type *"google.golang.org/genproto/googleapis/cloud/secretmanager/v1beta1".ListSecretVersionsRequest in argument to c.secretManagerClient.ListSecretVersions
../../../pkg/mod/github.com/!google!cloud!platform/berglas@v0.5.0/pkg/berglas/read.go:127:66: cannot use &secrets.GetSecretVersionRequest literal (type *secrets.GetSecretVersionRequest) as type *"google.golang.org/genproto/googleapis/cloud/secretmanager/v1beta1".GetSecretVersionRequest in argument to c.secretManagerClient.GetSecretVersion
../../../pkg/mod/github.com/!google!cloud!platform/berglas@v0.5.0/pkg/berglas/read.go:140:68: cannot use &secrets.AccessSecretVersionRequest literal (type *secrets.AccessSecretVersionRequest) as type *"google.golang.org/genproto/googleapis/cloud/secretmanager/v1beta1".AccessSecretVersionRequest in argument to c.secretManagerClient.AccessSecretVersion
../../../pkg/mod/github.com/!google!cloud!platform/berglas@v0.5.0/pkg/berglas/update.go:142:58: cannot use &secrets.GetSecretRequest literal (type *secrets.GetSecretRequest) as type *"google.golang.org/genproto/googleapis/cloud/secretmanager/v1beta1".GetSecretRequest in argument to c.secretManagerClient.GetSecret
../../../pkg/mod/github.com/!google!cloud!platform/berglas@v0.5.0/pkg/berglas/update.go:159:61: cannot use &secrets.CreateSecretRequest literal (type *secrets.CreateSecretRequest) as type *"google.golang.org/genproto/googleapis/cloud/secretmanager/v1beta1".CreateSecretRequest in argument to c.secretManagerClient.CreateSecret
../../../pkg/mod/github.com/!google!cloud!platform/berglas@v0.5.0/pkg/berglas/update.go:159:61: too many errors
exit status 1
```

I have manually upgraded everything in my go.mod except for berlgas, and upgrading berlgas changes nothing but the single berglas version line (plus adding a uuid library), so I think this is fairly isolated to berglas itself.

**Comment by sethvargo** [comment]
Are you depending on the Secret Manager library as a transitive dependency? It _should_ be pulling the version in go.mod, but it looks like it's pulling latest. I just opened 100 which uses the new import path.

**Comment by sethvargo** [comment]
Released 0.5.1 (it's currently building and will be available soon)

**Comment by josharian** [comment]
0.5.1 works for me. Thanks!
